examples/double-conversion/main.cpp

```diff
@@ -0,0 +1,6 @@
+#include <double-conversion/double-conversion.h>
```

Comment:
Looks like `*Config.cmake` is not correct:
```
fatal error: double-conversion/double-conversion.h: No such file or directory
```

Reply:
Not sure what you mean. It works for me using this command:

`TOOLCHAIN=default PROJECT_DIR=examples/double-conversion ./jenkins.py`

Reply:
> It works for me using this command

I can reproduce error on Travis:
* https://travis-ci.org/ingenue/hunter/builds/271825246?utm_source=github_status&utm_medium=notification

Reply:
Then why does only one build fail [here](https://travis-ci.org/ruslo/hunter/builds/271757246)?

Reply:
That build failure I referenced is actually a documentation error. Fixed in the change I just pushed.

Reply:
> Then why does only one build fail here?

It's different testing - only unit-tests and documentation, `example/double-conversion` is not tested.

Reply:
You are right. I had double-conversion installed locally, masking the error. Thanks. Will fix soon.